splice_spark/pom.xml

```diff
@@ -404,6 +404,82 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>dbaas</id>
+            <properties>
+                <scala.version>2.11.8</scala.version>
```

Comment:
`dbaas` profile is configured in multiple places (begs for refactoring) and `scala.version` is dragged along. Why don't we have a "top-level" profile with all the shared properties. I'm concerned that it could lead to version mismatch eventually.

Reply:
I declared it in this profile since there actually is a version mismatch between the dbaas service versus the on-prem service due to differing dependencies. <scala.version> declared in line 26 is overwritten when activating the profile.

Reply:
Why is this then in two places?! Here and in `scala_util/pom.xml`? Why is this not part of the top-level `dbaas` profile in `pom.xml` (that this PR just defined)? I must be missing something...